omod/src/main/java/org/openmrs/web/controller/OptionsFormController.java

```diff
@@ -203,7 +197,7 @@ protected ModelAndView onSubmit(HttpServletRequest request, HttpServletResponse
 			} else if (!"".equals(opts.getSecretAnswerNew())) {
 				// if they left the old password blank but filled in new
 				// password
-				errors.rejectValue("secretQuestionPassword", "error.password.incorrect");
```

Comment:
What is the reason behind changing this from error.password.incorrect to error.password.enter?

Reply:
I intended the output for that to be enter password rather than password incorrect

Reply:
@dkayiwa  I see that the code change here is not so effective after testing it via the RefApp, am going to change this